test/mediawiki/page_test.rb

```diff
@@ -208,3 +228,91 @@
     end
   end
 end
+
+describe 'ReplaceableContent' do
+  describe 'no whitespace is found after the template' do
+    let(:client) do
+      client = MiniTest::Mock.new
+      client.expect(
+        :get_wikitext,
+        FakeResponse.new(WIKITEXT_NO_WHITESPACE_AFTER_TEMPLATE),
+        ['Some Wiki page']
+      )
+      client
+    end
+    let(:section) do
+      MediaWiki::Page::ReplaceableContent.new(
+        client: client,
+        title: 'Some Wiki page',
+        template: 'Politician scraper comparison'
+      )
+    end
```

Comment:
Great, that''s much nicer. I'll just merge that commit as well if you're OK with that?